### ISSUE TITLE
Add a file to manticore-extra deb package

### ISSUE THREAD

**Issue by sanikolaev** [manticoresoftware/executor#64]
Let's include this file in the `manticore-extra` deb package to prevent `apt` from removing it, which could also lead to the removal of `manticore-columnar-lib`, `manticore-executor`, and `manticore-galera`.

```
root@9e8861525da4:/usr/share/manticore# cat /usr/share/manticore/manticore-extra.notice
manticore-extra is a virtual package that helps install various other Manticore components.
```

**Comment by djklim87** [comment]
Done since [v1.1.19](https://github.com/manticoresoftware/executor/tree/refs/tags/v1.1.19)

**Comment by sanikolaev** [comment]
Reopening since this looks wrong (called LICENSE, but is not a license):

```
[root@e808851ea0e9 /]# cat /usr/share/licenses/manticore-extra-1.1.19_24100311.b512d50/manticore-extra-LICENSE
manticore-extra is a virtual package that helps install various other Manticore components.
```

I'm not sure we needed to change anything in the rpm package at all.

**Comment by djklim87** [comment]
fixed in https://github.com/manticoresoftware/executor/tree/refs/tags/v1.1.20

**Comment by donhardman** [comment]
After this task CI is failing, we should fix it

**Comment by djklim87** [comment]
No. As I can see, the master was built without any errors https://github.com/manticoresoftware/executor/commit/8254967b42021a9449befa6bfa5f3be63b79ec9a


My commit also passed with no errors. But after you was returned it and canceled, it recognizing as failed

<img width="429" alt="image" src="https://github.com/user-attachments/assets/d03c8067-7b35-49de-98e2-f30627932f46">
